frontend/apps/opdscatalog/opdscatalog.lua

```diff
@@ -54,7 +54,7 @@ end
 
 function OPDSCatalog:onCloseWidget()
     UIManager:setDirty(nil, function()
-        return "partial", self[1].dimen
+        return "ui", self[1].dimen
```

Comment:
The FM *always* uses UI, so, doesn't affect the thumbnails any more than usual ;).

I *think* I may have originally switched those to partial at a time when I also had attempted to switch the FM to partial.
That didn't last long, though, and I probably forgot about the rest because it's mostly harmless ;).